boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.dts

```diff
@@ -204,9 +204,10 @@ ipc0: &cpuapp_cpurad_ipc {
 	hw-flow-control;
 };
 
-&pwm120 {
+&pwm130 {
 	status = "okay";
-	pinctrl-0 = <&pwm120_default>;
-	pinctrl-1 = <&pwm120_sleep>;
+	pinctrl-0 = <&pwm130_default>;
+	pinctrl-1 = <&pwm130_sleep>;
 	pinctrl-names = "default", "sleep";
+	memory-regions = <&cpuapp_dma_region>;
```

Comment:
Why is this not corrected in the first commit instead?

Reply:
Moved to first commit